### PR TITLE
Added GIT_COMMIT_HASH_SHORT

### PR DESCRIPTION
**Pull request by H3rmt**
Added GIT_COMMIT_HASH_SHORT to get short version of Head's hash

### CODE REVIEW

File: src/util.rs [hunk 1]
```diff
@@ -107,7 +107,7 @@ pub fn get_repo_description(root: &std::path::Path) -> Result<Option<(String, bo
 #[cfg(feature = "git2")]
 pub fn get_repo_head(
     root: &std::path::Path,
-) -> Result<Option<(Option<String>, String)>, git2::Error> {
+) -> Result<Option<(Option<String>, String, String)>, git2::Error> {
```
Comment (lukaslueg) [inline src/util.rs:110]: The docs need a slight update: "head's reference-name, long-hash and short-hash"

**Comment by lukaslueg** [comment]
Could we have a test [here](https://github.com/lukaslueg/built/blob/89683f1d2965d1ea2d14249748673777d86956b4/tests/testbox_tests.rs#L284) that demonstrates that the short hash and the long hash are both a `Some` and that the short hash is the prefix of the long hash?

**Comment by H3rmt** [comment]
added test in testbox testing and in-file testing to check is_some and starts_with

**Comment by lukaslueg** [comment]
Thanks

**Pull request merged**